Lib/pty.py

```diff
@@ -136,9 +130,32 @@ def _copy(master_fd, master_read=_read, stdin_read=_read):
     Copies
             pty master -> standard output   (master_read)
             standard input -> pty master    (stdin_read)"""
-    fds = [master_fd, STDIN_FILENO]
-    while fds:
-        rfds, _wfds, _xfds = select(fds, [], [])
+    os.set_blocking(master_fd, False)
```

Comment:
Would you mind commenting why this is necessary?

And also, even though `_copy` is a private function here, it is for sure used by end users in their programs (per Hyrum's Law) so it would be good to unset non-blocking when exiting `_copy` so that backward's compatibility is preserved.

Reply:
Changed as requested. PTAL